node/src/main/scala/coop/rchain/node/api/WebApi.scala

```diff
@@ -130,6 +134,9 @@ object WebApi {
 
     def isFinalized(hash: String): F[Boolean] =
       BlockAPI.isFinalized(hash).flatMap(_.liftToBlockApiErr)
+
+    def getTransaction(hash: String): F[TransactionResponse] =
+      cacheTransactionAPI.getTransaction(hash)
```

Comment:
Web API also needs to be typed and parsing errors should be moved to protocol level. Also task for API redesign.